packages/backend-core/src/platform/tenants.ts

```diff
@@ -99,3 +101,19 @@ export async function removeTenant(tenantId: string) {
     throw err
   }
 }
+
+export async function getTenantCreatedAt(tenantId: string) {
+  if (isSelfHostUsingCloud()) {
+    // we have no way of knowing when self-host tenants were created, so we
+    // can't return a createdAt date
+    return undefined
```

Comment:
As mentioned in here, if we store the version where they were created, we could solve this

Reply:
What do you mean?

Reply:
Sorry, here is [here](https://github.com/Budibase/budibase-pro/pull/394#discussion_r2053620813)